SPECS/lld/lld.spec

```diff
@@ -15,7 +15,7 @@ Source1:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%
 BuildRequires:  build-essential
 BuildRequires:  cmake
 BuildRequires:  file
-BuildRequires:  llvm-devel
+BuildRequires:  llvm-devel < 16.0.0
```

Comment:
There's also `lldb` suffering from trying to install both old and new version of LLVM.

Reply:
Thanks, I'll follow up on this separately if required